worlds/sm/__init__.py

```diff
@@ -682,6 +699,41 @@ def pre_fill(self):
                 loc.place_locked_item(item)
                 loc.address = loc.item.code = None
 
+    def post_fill(self):
```

Comment:
post_fill runs in parallel to other worlds' post fill and before progression balancing. Just making sure you know.

Reply:
Yes, it should be good.